test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+cargo test --lib -- --nocapture --include-ignored ::anchor
```

Comment:
Also include `set -eux` as the first instruction. Otherwise, cargo test failing won’t fail the test.

Reply:
got it